install_config/aggregate_logging.adoc

```diff
@@ -159,6 +159,9 @@ or template parameters (which are passed to the deployer in
 environment variables). The deployer looks for each value first in a
 *logging-deployer* ConfigMap, then a *logging-deployer* secret, then as
 an environment variable. Any or all may be omitted if not needed.
+Please note that if you are specifying values within a ConfigMap, the values
+will not be reflected in the output from `$oc new-app` but will still precede
```

Comment:
Remove the $ here. We tend to only include it in full code block examples.
Also, comma before but